libs/python/helperApiTest.py

```diff
@@ -0,0 +1,178 @@
+import requests
+import json
+import logging
+import sys
+import os
+from os.path import exists
+
+if not os.path.exists("./log"):
+    os.mkdir("./log")
+
+file_handler = logging.FileHandler(filename='./log/crossconsumption.log', mode='w')
+stdout_handler = logging.StreamHandler(sys.stdout)
+handlers = [file_handler, stdout_handler]
+
+logging.basicConfig(
+    level=logging.INFO,
+    format='%(asctime)s:%(levelname)s:%(message)s',
+    handlers=handlers
+)
```

Comment:
Logging defaults are defined in the btp_cli.py - why do you need these overrides?